model/src/main/java/org/eclipse/rdf4j/model/util/Models.java

```diff
@@ -65,31 +64,30 @@ public static Value anyObject(Model m) {
 	}
 
 	/**
-	 * Retrieves an object {@link Literal} value from the statements in the given model. If more than one
-	 * possible Literal value exists, any one Literal value is picked and returned.
+	 * Retrieves an object {@link Literal} value from the statements in the given model. If more than one possible
+	 * Literal value exists, any one Literal value is picked and returned.
 	 *
-	 * @param m
-	 * 		the model from which to retrieve an object Literal value.
-	 * @return an object Literal value from the given model, or {@link Optional#empty()} if no such value
-	 * exists.
+	 * @param m the model from which to retrieve an object Literal value.
+	 * @return an object Literal value from the given model, or {@link Optional#empty()} if no such value exists.
 	 */
 	public static Optional<Literal> objectLiteral(Model m) {
-		return m.stream().map(st -> st.getObject()).filter(o -> o instanceof Literal).map(
-				l -> (Literal)l).findAny();
+		return m.stream().map(st -> st.getObject()).filter(o -> o instanceof Literal).map(l -> (Literal) l).findAny();
```

Comment:
Here is an example of the formatter fixing where the old format did a line break in the wrong place.

Question. What happens if I format it myself as: 

```
return m
    .stream()
    .map(st -> st.getObject())
    .filter(o -> o instanceof Literal)
    .map(l -> (Literal) l)
    .findAny();

```

Will the formatter turn that back into a single line?

Reply:
@jeenbroekstra I was just a couple of seconds late. Do you know how the above would work?

Reply:
It will _only_ turn it back into a single line if it would actually fit in 120 chars. Otherwise it would reformat to something _almost_ identical to what you have here. This is actually the case I was talking about when I mentioned "qualified invocations": it's been modified to put each successive call on a new line (at least if it needs linebreaks), instead of just chaining the whole thing.

Reply:
I just gave it a quick go and how it formats is as follows. Your original line is < 120 chars so it would just fit it on one line:

    return m.stream().map(st -> st.getObject()).filter(o -> o instanceof Literal).map(l -> (Literal) l).findAny();


But if we tweak it slightly to make it too long, it would reformat like this:

		return modelWithAVeryLongName.stream()
				.map(st -> st.getObject())
				.filter(o -> o instanceof Literal)
				.map(l -> (Literal) l)
				.findAny();

Reply:
I would like it to be a one way thing. Long one-liners get split, but already multiline formatted fluent api stays multiline.